dask/values.yaml

```
@@ -131,6 +131,76 @@ worker:
       podTargetLabels: [] # PodTargetLabels transfers labels on the Kubernetes Pod onto the target.
       metricRelabelings: [] # MetricRelabelConfigs to apply to samples before ingestion.
 
+additional_worker_groups:
+  - worker:
+      name: worker  # Dask worker name.
```

The `name` in the example should also be different to the name of the main worker group. 
```suggestion
      name: high-mem-workers  # Dask worker group name.
```